src/components/tl/nccl/tl_nccl_coll.h

```diff
@@ -44,6 +44,10 @@ ucc_status_t ucc_tl_nccl_alltoallv_init(ucc_tl_nccl_task_t *task);
 
 ucc_status_t ucc_tl_nccl_bcast_init(ucc_tl_nccl_task_t *task);
 
+ucc_status_t ucc_tl_nccl_send_init(ucc_tl_nccl_task_t *task);
+
+ucc_status_t ucc_tl_nccl_recv_init(ucc_tl_nccl_task_t *task);
+
```

Comment:
I think these can be dropped. They were part of my PR that we don't need anymore.